tests/test_methods.py

```diff
@@ -268,8 +268,13 @@ def test_save_intermediate_data_frames_per_chunk(
     with h5py.File(tmp_path / FILE_NAME, "r") as f:
         chunk_shape = f[DATA_PATH].chunks
 
-    if frames_per_chunk != 0:
+    if frames_per_chunk > 0:
         assert chunk_shape == tuple(expected_chunk_shape)
+    elif frames_per_chunk == -1:
+        # when this is -1, it is decided autmatically and because of the
+        # very small frame size, this exceeds the value of the slicing
+        # dimension, and the fallback is 1
```

Comment:
Is this note about how small frame sizes can influence the chunk size when automatically deciding the chunk size worth putting in the function doing the chunk size calculation, or is it only applicable to the test?

I'm basically wondering if this is a situation that could crop up often enough that it's worth having the info in the function doing the calculation, rather than tucked away in a test.

Reply:
> For this to happen the frame will need to be several order of magnitudes smaller than 512 MiB

Gotcha, thanks for the clarification. It does seem like this information is relevant only in testing cases - which means it makes sense to leave in the test.